spec/factories/ext_management_system.rb

```diff
@@ -327,7 +327,9 @@
           :parent  => :configuration_manager
 
   trait(:provider) do
-    after(:build, &:create_provider)
+    after(:build) do |x|
```

Comment:
Sounds like the commit message specifically explains the bug:  https://github.com/ManageIQ/manageiq/pull/13104/commits/d94a4002c7facf9f2d11fadd822a5b0c81dc9092